tests/Transformer/YamlTransformerTest.php

```diff
@@ -137,7 +137,7 @@ public function testNoHeader()
 YAML;
 
         $file = $this->getFilePath('config.yml');
-        file_put_contents($file, $content);
+        file_put_contents($file ?? '', $content);
```

Comment:
Whether this is null or an empty string, an error is thrown anyway:
> Uncaught ValueError: Path cannot be empty

Is there a deprecation warning _as well_ as the error if it's a null?

Reply:
```
php -a
php > file_put_contents(null, 'abc');
PHP Deprecated:  file_put_contents(): Passing null to parameter #1 ($filename) of type string is deprecated in php shell code on line 1

Deprecated: file_put_contents(): Passing null to parameter #1 ($filename) of type string is deprecated in php shell code on line 1
PHP Warning:  Uncaught ValueError: Path cannot be empty in php shell code:1
Stack trace:
#0 php shell code(1): file_put_contents()
#1 {main}
  thrown in php shell code on line 1

Warning: Uncaught ValueError: Path cannot be empty in php shell code:1
Stack trace:
#0 php shell code(1): file_put_contents()
#1 {main}
  thrown in php shell code on line 1
```